hokusai/auct-281.yml

```diff
@@ -0,0 +1,149 @@
+---
```

Comment:
Did you mean to commit this file?

Reply:
Want to delete it before merging the PR, just wanted to show reviews an example of a fulled processed review app.